zerver/decorator.py

```diff
@@ -18,7 +18,10 @@
     is_remote_server
 from zerver.lib.exceptions import RateLimited
 from zerver.lib.rate_limiter import incr_ratelimit, is_ratelimited, \
-    api_calls_left
+    api_calls_left, USER_PREFIX, is_ratelimited_ip, incr_ratelimit_ip, api_calls_left_ip, \
+    IP_TYPE, IP_PREFIX, RATELIMIT_APPLIED_LIMITS, RATELIMIT_SECS_TO_FREEDOM, \
+    RATELIMIT_OVER_LIMIT, RATELIMIT_REMAINING, is_ratelimited_email, \
+    incr_ratelimit_email, EMAIL_TYPE, EMAIL_PREFIX, api_calls_left_email
```

Comment:
It feels wrong to me that we need to import so many constants here.  Is there a way to have some of the logic happen in rate_limiter, or have a config object or something?  This is not urgent, just feels like a smell.